### PR TITLE
New post: Why Rust?

### PR DESCRIPTION
**Pull request by llogiq**
Here's the article. Perhaps someone finds a better preview image? I just took the official Rust logo as a placeholder.

### CODE REVIEW

File: qdrant-landing/content/articles/why-rust.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+| Engine   | setup_name             | Dataset               | Upload Time (s) | Upload + Index Time (s) | P95 (s) | RPS     | P99 (s) | Latency (s) | Precision | engine_params                    |
+|----------|------------------------|-----------------------|-----------------|-------------------------|---------|---------|---------|------------–|-----------|----------------------------------|
+| qdrant   | qdrant-rps-m-64-ef-512 | deep-image-96-angular | 845.78          | 8959.44                 | 0.055   | 1541.86 | 0.063   | 0.024       | 0.96      | {"search_params":{"hnsw_ef":64}} |
+| weaviate | weaviate-m-16-ef-128   | deep-image-96-angular | 8922.25         | 8922.25                 | 0.35    |  507.33 | 0.41    | 0.19        | 0.94      | {"vectorIndexConfig":{"ef":256}} |
+
+(Please note that the performance difference is not only down to the languages used, the implementations differ in algorithms and design tradeoffs. As always, use your own best judgement.)
```
Comment (generall) [inline qdrant-landing/content/articles/why-rust.md:36]: let's remove, maybe? I don't think the main factor is the language in this particular case

**Comment by llogiq** [comment]
Adressed the comments here and on slack. https://deploy-preview-135--condescending-goldwasser-91acf0.netlify.app/articles/why-rust/ now shows the preview image (although we may want to go more fancy?) and I removed the benchmark table.

**Review by kacperlukawski — CHANGES_REQUESTED** [review]
It would be good to introduce more headers in the text, as it would be easier to follow.

**Review by kacperlukawski — APPROVED** [review]

**Comment by kacperlukawski** [comment]
We can just use a picture generated by Midjourney as a preview image

**Pull request merged**